app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -14,7 +14,33 @@ class Constants
         code: 6602,
         label: 'asthma',
         flipper_name: 'asthma',
-        sidekiq_job: 'RapidReadyForDecision::Form526AsthmaJob'
+        sidekiq_job: 'RapidReadyForDecision::Form526AsthmaJob',
+        keywords: %w[
```

Comment:
For future reference: https://www.notion.so/Asthma-meds-filter-for-PDF-da5f27c343f24e3daf5e08c05c2d9639